File: leetcode/21. countGoodTriplets/countGoodTriplets.go
package countgoodtriplets

import (
	"math"
)

func countGoodTriplets(arr []int, a int, b int, c int) int {
	n := len(arr)
	res := 0
	for i := 0; i < n-2; i++ {
		for j := 0; j < n-1; j++ {
			for k := 0; k < n; k++ {
				va, vb, vc := arr[i], arr[j], arr[k]
				if int(math.Abs(float64(va-vb))) <= a && int(math.Abs(float64(vb-vc))) <= b && int(math.Abs(float64(va-vc))) <= c {
					res++
				}
			}
		}
	}
	return res
}

File: leetcode/21. countGoodTriplets/countGoodTriplets.py
from typing import List

def countGoodTriplets(arr: List[int], a: int, b: int, c: int) -> int:
    n = len(arr)
    res = 0
    for i in range(n-2):
        for j in range(i+1, n-1):
            for k in range(j+1, n):
                va, vb, vc = arr[i], arr[j], arr[k]
                if abs(va-vb) <= a and abs(vb-vc) <= b and abs(va-vc) <= c:
                    res += 1
    return res